tests/unit/modules/test_esxi.py

```diff
@@ -10,6 +10,8 @@
 import pytest
 import saltext.vmware.modules.esxi as esxi
 import saltext.vmware.utils.connect
+from config_modules_vmware.schema.schema_utility import Product
```

Comment:
The lines could be as follows:
```
from config_modules_vmware.schema.schema_utility import Product, retrieve_reference_schema

Reply:
As discussed resolving this comment, since pre-commit is splitting these imports into separate lines.

```
Reorder python imports...................................................Failed
- hook id: reorder-python-imports
- exit code: 1
- files were modified by this hook

Reordering imports in tests/unit/modules/test_esxi.py
Reordering imports in src/saltext/vmware/modules/esxi.py
```